tests/cbmc/proofs/s2n_stuffer_printf/s2n_stuffer_printf_harness.c

```diff
@@ -23,6 +23,11 @@
 
 int nondet_int(void);
 
+/* The MacOS.sdk defines a macro called "vsnprintf" which breaks the
+ * function definition below, so make sure it's undef'd here.
+ */
+#undef vsnprintf
```

Comment:
Same-- no macos testing, so we have no way to actually know that this fix works.

Reply:
It does work because I've been running all the proofs on macOS for weeks. Honestly - it does!

Reply:
Yeah I guess it's fine to leave these. They're only in the CBMC proofs, so they're not complicating the source code.